.github/workflows/miniconda.yml

```diff
@@ -18,29 +18,35 @@ jobs:
         exclude:
           - os: macos-latest  
             platform: x32
+      fail-fast: false
+
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
-    - name: Setup Conda
-      uses: s-weigand/setup-conda@v1
+    - name: Setup Micromamba
+      uses: mamba-org/provision-with-micromamba@v13
       with:
-        activate-conda: false
-        conda-channels: conda-forge
+        environment-file: false
 
     - name: Python ${{ matrix.python-version }}
       shell: bash -l {0}
       run: |
-        conda create --name TEST python=${{ matrix.python-version }} numpy cython pip pytest hdf5 libnetcdf cftime --strict-channel-priority
-        source activate TEST
+        micromamba create --name TEST python=${{ matrix.python-version }} numpy cython pip pytest hdf5 libnetcdf cftime "zlib=1.2.11" --channel conda-forge
```

Comment:
```suggestion
        micromamba create --name TEST python=${{ matrix.python-version }} numpy cython pip pytest hdf5 libnetcdf cftime zlib --channel conda-forge
```
does this work?

Reply:
Deeper explanation; https://github.com/Unidata/netcdf4-python/issues/1186#issuecomment-1233339986